src/MSBuild/XMake.cs

```diff
@@ -1114,6 +1126,7 @@ string outputResultsCache
                     }
 
                     parameters.EnableNodeReuse = enableNodeReuse;
+                    parameters.LowPriority = lowPriority;
```

Comment:
I'd rename this to something else than `BuildParameters.LowPriority`, because it looks like there might be multiple intents to playing around with priority, and the actual process/thread priority for one intent might be different then the other. How about making it a flag enum instead, something like `BuildParameters.PriorityMode`, where one value might be ``UserLowPriority`, another future one might be `NodeHandshake`. The good part is that we can keep adding new values without changing the public API.

Reply:
Is there any reason not to reuse the ProcessPriorityClass enum?

Also, since there is no longer another intention to change priority in the immediate future, I'm thinking it's better to leave it as LowPriority. It's messier if we want to go in and change it again, but I started making it fully generic and ran into difficulties with ShutdownAllNodes, since if it can take any value in PriorityClass, I'd either need a hash function that allocates specific bits (or equivalent) for the priority to make it easy to generate all relevant priorities or accept that I'll miss some nodes or shut down too many. Seems like too much headache for more generality that we might not use. I'll still make the `PriorityUtils.SwitchProcessPriorityTo` take a ProcessPriorityClass so it can be reasonably generic.